esp32.c

```diff
@@ -14,9 +14,10 @@
 #include <time.h>
 #include <unistd.h>
 #include <sqlite3.h>
-#include <esp_spi_flash.h>
+#include <spi_flash_mmap.h>
 #include <esp_system.h>
 #include <esp_random.h>
+#include <esp_timer.h>
```

Comment:
I think this is just used for IO_STATS so maybe better:

```suggestion
#ifdef DEBUG_IO_STATS
#include <esp_timer.h>
#endif
```

Of course moving the `#define DEBUG_IO_STATS` with comment on the top?

Reply:
Done